_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -122,10 +122,61 @@ This `iframe` element contains a link that is not part of its [sequential focus
 <iframe tabindex="-1" srcdoc="<a href='/' tabindex='-1'>Home</a>"></iframe>
 ```
 
+#### Inapplicable Example 5
+
+This `iframe` element is [inert][] because of its own `inert` [attribute value][].
+
+```html
+<iframe inert srcdoc="<a href='/'>Home</a>"></iframe>
+```
+
+#### Inapplicable Example 6
+
+Once "Privacy policy details" button is activated, the `iframe` element become [inert][] because of showModal() method that causes the `iframe` to be [blocked by a modal].
```

Comment:
I think we should make the example so that the modal is shown at load time.
Currently, examples do not require manipulation before testing. There are some steps in that direction as part of the "states" discussion (#1953, #2045).

In this case, we can simply call the `showModal` in an `onload` call to ensure this is testable out of the box. I think the example will still look "real" enough that people will understand why the modal is opened by default. (otherwise a more realstic case could be to have a cookies banner or region selection modal that is opened on page load and blocking interaction with the rest of the page).